README.md

```diff
@@ -1,26 +1,13 @@
-![jwt-auth-banner](https://cloud.githubusercontent.com/assets/1801923/9915273/119b9350-5cae-11e5-850b-c941cac60b32.png)
+## Credits
+[This repository it's a fork from original tymonsdesigns/jwt-auth](https://github.com/tymondesigns/jwt-auth/wiki), we decided to fork and work idenpdendent because the original one was not being updated for long time and keep doing support for the application.
```

Comment:
`is` instead `it's`
`idenpdendent` should be `independent`

Reply:
I did the changes.